.pylintrc

```diff
@@ -523,7 +523,7 @@ max-returns=6
 max-statements=50
 
 # Minimum number of public methods for a class (see R0903).
-min-public-methods=2
+min-public-methods=1
```

Comment:
Ok so I did some further review of your program and proper programming practices, and I made the conclusion that we should keep this at 2 minimum public methods for a class.